ipfs-api-prelude/src/api.rs

```diff
@@ -10,7 +10,7 @@ use crate::{read::LineDecoder, request, response, Backend, BoxStream};
 use async_trait::async_trait;
 use bytes::Bytes;
 use common_multipart_rfc7578::client::multipart;
-use futures::{future, FutureExt, TryStreamExt, AsyncRead};
+use futures::{future, AsyncRead, FutureExt, TryStreamExt};
```

Comment:
We need `rustfmt.toml`

cc @ferristseng, what do you think?

Reply:
Yes! I think that is a good idea

Reply:
Addressed in https://github.com/ferristseng/rust-ipfs-api/pull/105